### PR TITLE
Accessory Girl(west town)(Ready)

### PR DESCRIPTION
**Pull request by PummelsTL**

### CODE REVIEW

File: CSV/Language/English/Place/WestTown.txt [hunk 1]
```diff
@@ -16,146 +16,146 @@ Place/WestTown,
...
+これじゃ売りものにできないわよねー。\n当然、弁償してもらわないと♪,There's no way I can sell it now. And that means... You'll have to pay for it♪
+そう言って女は手の平を差し出した。早くお金を寄越せということらしい。,The woman put out her hand after saying those words. Apparently; she wanted me to pay right away.
+ううっ・・・。,Uuuh...
+あんな状態にした指輪を前に、言い訳することもできず、僕は大金を失ってしまったのだった。,Nothing I said would get me out of this when the ring was covered in my semen. I gave the woman the 8000G she wanted.
+くす、そんな指輪、もう使い道ないと思うけど、\nたまに見てわたしとのエッチ思い出してね❤,Ehehe❤ I doubt the ring is going to be useful anymore; but at least; you can keep it as a memento of my boobjob❤
+もしムラムラして我慢できなくなったら、\nおっぱいでまたあなたのお金搾り取ってあげる❤,And if looking at it makes you so horny that you can't take it; just come to me and I'll squeeze your wallet dry with my breasts again❤
+彼女は胸をむにゅむにゅと揺らし、さっきまでのパイズリを思い出させる動きを僕に見せつけてきた。,The woman shook her breasts up and down in the air to demonstrate how she gave me a boobjob.
+僕はさっき射精したばかりだというのに、エロ過ぎる光景を前にまた前かがみになってしまう。,That sight was so lewd that I couldn't help getting another erection and slouching over despite ejaculating just now.
+ま、さすがに3回も貢ぎに来るほどバカじゃないわよね？\nそれじゃあねー。,But no one would be as stupid as to fall for the same trick three times; right? Bye bye.
```
Comment (OfficialLayton) [inline CSV/Language/English/Place/WestTown.txt:148]: "Bye bye." might seem a little lacking in emotion comapred to "それじゃあねー"

Maybe "Catch you later-", "See ya-"?

Comment (PummelsTL) [reply CSV/Language/English/Place/WestTown.txt:148]: I find a general bye more fitting since she's not expecting to see you again.

Comment (OfficialLayton) [reply CSV/Language/English/Place/WestTown.txt:148]: Hmm in that case "See ya-" might be more fitting then.

**Comment by OfficialLayton** [comment]
Hi @PummelsTL! Sorry about the late merge for this PR.

I've just done a regex find and replace to add '.' before the hearts. I'll merge it now. I apologise for the delay ^^

**Pull request merged**